app/teachingCall/teachingCallStatus/directives/modals/addInstructorsModal/addInstructorsModal.js

```diff
@@ -29,6 +29,10 @@ let addInstructorsModal = function (TermService, TeachingCallStatusActionCreator
         instructor.invited = false;
       });
 
+      scope.invitedInstructorsCount = function() {
+        return scope.startTeachingCallConfig.invitedInstructors.filter(function(el) { return el.invited == true; }).length;
```

Comment:
I'd recommend using 'instructor' for the iterator variable, generic variable names can make the code harder to read in the future